Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -92,6 +92,22 @@ void ShutdownSoundStream()
   INFO_LOG(AUDIO, "Done shutting down sound stream");
 }
 
+std::string GetDefaultSoundBackend()
+{
+  std::string backend = BACKEND_NULLSOUND;
+#if defined __linux__
+  if (AlsaSound::isValid())
```

Comment:
Yes, I just moved the same logic that was in the ConfigManager.
I think it would be better to have a sorted array of weighted backends and pick the first / last one. Or have a flag to mark them as default on the platform if found... Well, it's easy to iterate on this now, it's all in the same file!